examples/structure.txt

```diff
@@ -0,0 +1,36 @@
+output

```

Comment:
This documentation could be converted to Markdown and put in a README. A diagram would be helpful to understand the rendering process.